Project.toml

```diff
@@ -4,21 +4,22 @@ version = "0.4.6"
 
 [deps]
 Crayons = "a8cc5b0e-0ffa-5ad4-8c14-923d3ee1735f"
+ImageBase = "c817782e-172a-44cc-b673-b171935fbb9e"
 ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
-ImageTransformations = "02fcd773-0e25-5acc-982a-7f6622650795"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 
 [compat]
 Crayons = "0.5, 1, 2, 3, 4"
-ImageCore = "0.8.1, 0.9"
-ImageTransformations = "0.5, 0.6, 0.7, 0.8"
+ImageBase = "0.1"
+ImageCore = "0.9"
```

Comment:
`restrict` requires ColorVectorSpace to work. I bump the lower bound of ImageCore to 0.9 because it loads ColorVectorSpace https://github.com/JuliaImages/ImageCore.jl/pull/157